x-pack/plugins/cases/server/saved_object_types/user_actions.ts

```diff
@@ -60,6 +60,12 @@ export const createCaseUserActionSavedObjectType = (
               persistableStateAttachmentTypeId: { type: 'keyword' },
             },
           },
+          assignees: {
+            properties: {
+              // assignees.uid
+              uid: { type: 'keyword' },
```

Comment:
why this change, is this another one of those properties that were already saved in the payload we just didn't have to explicitly access it?

Reply:
To do a `terms` aggregation the field should be indexed. Because it is not in the mapping it is not indexed. By adding it to the mapping the field will be indexed.

Reply:
Ah, thank you for clarifying 👌